tutorials/3d/introduction_to_3d.rst

```diff
@@ -190,22 +190,22 @@ light to your objects.
 Cameras
 -------
 
-No matter how many objects are placed in 3D space, nothing will be
+No matter how many objects are placed in the 3D space, nothing will be
 displayed unless a :ref:`Camera <class_Camera>` is
-also added to the scene. Cameras can either work in orthogonal or
+also added to the scene. Cameras can work in either orthogonal or
 perspective projections:
 
 .. image:: img/tuto_3d10.png
 
-Cameras are associated with and only display to a parent or grandparent
+Cameras are associated with -- and only display to -- a parent or grandparent
```

Comment:
Shouldn't this use em dashes (`—`)?

Nevertheless, I think commas or parentheses would work just a well here, and are probably better understood by non-native speakers than em dashes (as the latter aren't really idiomatic in most languages (at least, in those I know)).

Reply:
@akien-mga : I've had a similar conversation with @bojidar-bg about em dashes. The concern I raised then was that some readers' devices might not properly render the Unicode em dash character.
At any rate, I suppose I could use parentheses in this case.

Reply:
I think all browsers and OSes currently supported on the market have decent support for Unicode. Emojis made sure of that :)

But yes, I think parentheses are "safer" here.

Reply:
@akien-mga : I try not to marginalize Luddites.